Formula/racer.rb

```diff
@@ -0,0 +1,33 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.1.0.tar.gz"
+  sha256 "f969e66d5119f544347e9f9424e83d739eef0c75811fa1a5c77e58df621e066d"
+  head "https://github.com/phildawes/racer.git"
+
+  depends_on "rust" => :build
+
+  resource "rust_source" do
+    url Formula["rust"].stable.url
+    sha256 Formula["rust"].stable.checksum.hexdigest
+  end
+
+  def install
+    system "cargo", "build", "--release"
+    (libexec/"bin").install "./target/release/racer"
+    (bin/"racer").write_env_script(libexec/"bin/racer", :RUST_SRC_PATH => "#{HOMEBREW_PREFIX}/share/rust_src/current")
```

Comment:
Should this not just be `#{share}/rust_src/current` rather than `HOMEBREW_PREFIX`? I tend to feel like `HOMEBREW_PREFIX` usage makes it sound the directory will exist across updates, which it won't here.

Apologies if I missed a discussion here. GH squashes "old" diffs which makes it hard to glance through everything.